cmd/serve/imagebundle/image_bundle.go

```diff
@@ -41,13 +42,23 @@ func NewCommand(out output.Output) *cobra.Command {
 
 			out.EndOperation(true)
 
-			out.StartOperation("Unarchiving image bundle")
-			err = archiver.Unarchive(imageBundleFile, tempDir)
-			if err != nil {
-				out.EndOperation(false)
-				return fmt.Errorf("failed to unarchive image bundle: %w", err)
+			sort.Strings(imageBundleFiles)
+
+			extractedBundles := make(map[string]struct{}, len(imageBundleFiles))
+
+			for _, imageBundleFile := range imageBundleFiles {
+				if _, ok := extractedBundles[imageBundleFile]; ok {
+					continue
+				}
```

Comment:
This is here just in case users provide the same bundle name twice?

Reply:
Yeah, it looked weird when I was outputting the `unarchiving ...` message to see the same file in there twice.

Reply:
I added a comment above explaining this.